src/server/chromium/crPage.ts

```diff
@@ -542,12 +544,33 @@ class FrameSession {
     this._page._frameManager.frameCommittedSameDocumentNavigation(frameId, url);
   }
 
+  private _takeParentForSwappingOutFrame(targetId: string) {
+    for (const frame of this._page.frames()) {
+      if (!(frame as any)[swappingOutChildFrames])
+        continue;
+      if ((frame as any)[swappingOutChildFrames].delete(targetId))
+        return frame._id;
+    }
+    return null;
+  }
+
+  private _frameMaybeSwappingOut(frameId: string) {
+    const frame = this._page._frameManager.frame(frameId);
+    if (!frame) return;
```

Comment:
style: return on the next line. I am surprised that eslint does not complain...

Reply:
yeah, I relied on it